src/packages/@ncigdc/containers/CasePage.js

```diff
@@ -72,11 +69,9 @@ export const CasePageComponent = ({ node, viewer }: TProps) => (
           query={ctx.query}
           node={node}
           totalFiles={node.files.hits.total}
-          ssmTested={_.get(
-            viewer.explore.cases.hits,
-            "edges[0].node.available_variation_data",
-            []
-          ).includes("ssm")}
+          ssmTested={(viewer.explore.cases.hits.edges[0].node
```

Comment:
well.. that is to say, it's failing currently at the route level because ES returns a 404 style error